.github/workflows/release.yaml

```diff
@@ -86,13 +86,9 @@ jobs:
       uses: extractions/setup-just@v2
 
     - name: "📗 Generate Documentation"
-      run: |
-        just gen-docs
-        git config user.name 'open-sauced[bot]'
-        git config user.email '63161813+open-sauced[bot]@users.noreply.github.com'
-        git add docs/
-        git commit -m "chore: automated docs generation for release" || echo "No changes to commit"
-        git push origin HEAD:${{ github.ref }}
+      run: ./scripts/generate-docs.sh
```

Comment:
I popped this into a script now as there is a but more happening now. The script has been `chmode +x`ed.